src/utils/db-handler.go

```diff
@@ -0,0 +1,43 @@
+package utils
+
+import (
+	"database/sql"
+	"fmt"
+
+	_ "github.com/lib/pq"
+)
+
+func GetDatabaseConnection() *sql.DB {
+	psqlconn := fmt.Sprintf("host=%s port=%d user=%s password=%s dbname=%s sslmode=disable",
+		GetEnv("POSTGRES_HOST", ""),
+		GetEnv("POSTGRES_PORT", ""),
+		GetEnv("POSTGRES_USER", ""),
+		GetEnv("POSTGRES_PASSWORD", ""),
+		GetEnv("POSTGRES_DB", ""))
+
+	db, err := sql.Open("postgres", psqlconn)
+	if err != nil {
+		panic(err)
+	}
+
+	defer db.Close()
```

Comment:
I think this line might cause your crash. The `defer` statement will close the db handle as soon as the function returns.

Reply:
You're right, the Error was: "sql: database is closed". Also the Port was handed over as string but parsed as digit.